radio/radio_test.ts

```diff
@@ -138,6 +138,20 @@ describe('<md-radio>', () => {
       expect(a2.element.checked).withContext('prev radio checked').toBeFalse();
     });
 
+    it('dispatched a change event on user navigation', async () => {
+      const {harnesses, root} = await setupTest(radioGroupPreSelected);
+      const changeHandler = jasmine.createSpy('changeHandler');
+      root.addEventListener('change', changeHandler);
+      const [, a2] = harnesses;
+      expect(a2.element.checked)
+          .withContext('default checked radio')
+          .toBeTrue();
+      
```

Comment:
nit: I think there's trailing whitespace on this line we'll need to fix

Reply:
done